docs/source/contributing/acceptance-tests.md

```diff
@@ -30,19 +30,19 @@ All sessions should start from the `packages/volto` directory.
 1.  In the first session, start the backend server.
 
     ```shell
-    make start-test-acceptance-server
+    make acceptance-backend-start
```

Comment:
@stevepiercy I think a note mentioning that they were changed from volto 18 would bring
more awareness. we do have notes about commands in other sections and given that this
is something that is required for a good pull request it's better to give the user a heads-up.
In the same route I feel that the upgrade guide regarding make commands should mention
what area might be affected by this change.
Something along the lines:
Most of these commands were used internally by Volto core for its development, however,
one change that will affect most users is the change regarding the performing of acceptance tests.
Have a look at the acceptance docs section for the new commands used.
@sneridagh what do you think about this?

Another thing that we could do is to have for this version some convenience declarations

start-test-acceptance-server:
   echo  "start-test-acceptance-server has been renamed, use:
       make acceptance-frontend-start"
It's only 3-4 commands and we give the user the ability to quickly do acceptance testing
without having to go and study everything that has changed.

Reply:
That's a bad example, because the command was documented incorrectly in the first place (it used to apply for CI), and was not a simple rename. Otherwise, yes, warnings of changes _may_ be appropriate. For this one case, I would not bother.

Victor and I've already invested too many hours in getting makefile commands unified, and we are both weary from it. It's tedious. I am "meh" on adding `echo` because developers can do `make help` to find the command they want to use, or consult the Upgrade Guide. I think it would be helpful, but I just don't have the energy to champion this enhancement.

Reply:
@ichim-david 

Regarding the first thing, we could add the deprecation notice. However, I tend to follow Steve suggestions on this regard. At the end, (unfortunately) the core developers are not counted by the thousands.

Regarding the last suggestion. I'm with Steve in this one. This is the PR from hell of the season, if someone wants to go the extra mile, please be my guest. IMHO we have better things to take care of. The users of this are core developers, we will manage.